src/Compilers/CSharp/Portable/Symbols/Retargeting/RetargetingSymbolTranslator.cs

```diff
@@ -57,9 +57,29 @@ private RetargetingNamedTypeSymbol CreateRetargetingNamedType(Symbol symbol)
             return new RetargetingNamedTypeSymbol(this, (NamedTypeSymbol)symbol);
         }
 
-        private RetargetingFieldSymbol CreateRetargetingField(Symbol symbol)
+        private FieldSymbol CreateRetargetingField(Symbol symbol)
         {
             Debug.Assert(ReferenceEquals(symbol.ContainingModule, _underlyingModule));
+            if (symbol is TupleErrorFieldSymbol tupleErrorField)
+            {
+                var correspondingTupleField = tupleErrorField.CorrespondingTupleField;
+                Debug.Assert(correspondingTupleField is TupleErrorFieldSymbol);
+
+                var retargetedCorrespondingDefaultFieldOpt = (correspondingTupleField == (object)tupleErrorField)
+                    ? null
+                    : (TupleErrorFieldSymbol)RetargetingTranslator.Retarget(correspondingTupleField);
+
+                return new TupleErrorFieldSymbol(
+                    RetargetingTranslator.Retarget(tupleErrorField.ContainingType, RetargetOptions.RetargetPrimitiveTypesByTypeCode),
```

Comment:
>RetargetOptions.RetargetPrimitiveTypesByTypeCode [](start = 83, length = 48)

```RetargetOptions.RetargetPrimitiveTypesByName``` is more appropriate here even though this shouldn't make any difference for a ValueTuple type. See how RetargetingFieldSymbol retargets ContainingSymbol.